### ISSUE TITLE
Remove VerifyOpenProjectFileCapability usage inside shell & dialog

### ISSUE THREAD

**Issue by davkean** [dotnet/project-system#4071]
This is no longer needed after #4066.

**Comment by davidwengier** [comment]
PR for this: https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/147429

**Comment by davidwengier** [comment]
PR has been merged
